src/software/ai/hl/stp/play/ball_placement/ball_placement_play_fsm.cpp

```diff
@@ -53,12 +126,12 @@ void BallPlacementPlayFSM::alignPlacement(const Update &event)
                 .normalize();
         Angle setup_angle = alignment_vector.orientation();
         setup_point       = event.common.world_ptr->ball().position() -
-                      2 * alignment_vector * ROBOT_MAX_RADIUS_METERS;
+                      2.5 * alignment_vector * ROBOT_MAX_RADIUS_METERS;
```

Comment:
`2.5 * ROBOT_MAX_RADIUS_METERS` should be a constant: `ALIGN_DISTANCE_FROM_BALL_M`